public/coneref/refbasics.html

```diff
@@ -18,14 +18,18 @@
 	such as this one which calculates the value of pi:</p>
 	
 <pre>
+import stdio::*
+
 fn main():
   imm pi = calcpi(10)
+	print <- "\u03C0=", calcpi(10), "\n" // <-- Change 10 to 2 for less accurate pi
```

Comment:
Would you check line indentation here and on new line 31. I get the impression tabs are injected here, rather than spaces, which will may not render correctly. Lines 24/25/31 should all have just two spaces of indentation.

Reply:
It only mattered for the <pre> blocks, because that is the only place where which one you use changes how it renders.  

In front of the rest of the HTML, whether one uses spaces or tabs makes no difference to browser rendering.